### PR TITLE
Move Consul startup out of rc.local and into /etc/nubis.d/00-consul

### LINKED ISSUE #258: Move consul startup to /etc/nubis.d/00-consul

### PR DESCRIPTION
**Pull request by gozer**
Fixes #258

### CODE REVIEW

File: nubis/puppet/init.pp [hunk 1]
```diff
@@ -1,14 +0,0 @@
...
-import 'consul.pp'
-import 'fluentd.pp'
-import 'jq.pp'
-import 'postfix.pp'
-import 'datadog.pp'
-import 'mig.pp'
-import 'credstash.pp'
-import 'consulate.pp'
-import 'nubis_lib.pp'
-import 'proxy.pp'
```
Comment (limed) [inline nubis/puppet/init.pp]: These removals don't seem to have anything to do with this PR, other than that it looks good

File: nubis/puppet/files/rc.local [hunk 2]
```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
```
Comment (limed) [inline nubis/puppet/files/rc.local]: Does it need a `service confd restart` here to replicate what you removed in consul-autojoin.sh?

File: nubis/puppet/init.pp (hunk 1 above)
Comment (gozer) [inline nubis/puppet/init.pp]: Yeah, that was just a drive-by cleanup

File: nubis/puppet/files/rc.local (hunk 2 above)
Comment (gozer) [inline nubis/puppet/files/rc.local]: Good catch, I removed that bit on purpose, because it was redundant. confd's own startup takes place after consul has started (in /etc/nubis.d/confd) and does this already.

**Comment by limed** [comment]
r+
